### ISSUE TITLE
Expand Testing Guides to include Cypress

### ISSUE THREAD

**Issue by crutchcorn** [withastro/docs#2168]
Currently, [our testing guide page](https://docs.astro.build/en/guides/testing/) states the following:

> Testing helps you write and maintain working Astro code. Astro supports many popular tools for unit tests, component tests, and end-to-end tests including Jest, Mocha, Jasmine, Cypress and Playwright.

However, the page only has a mention for how to integrate Playwright into E2E testing for Astro apps.

It would be awesome to see similar content for Cypress, since it's explicitly mentioned as well.

**Comment by sarah11918** [comment]
Yes! We would love to have more testing info expanded to incorporate different testing platforms!

This page was created in response to a community submission specifically to document their own Playwright setup. I built up a fuller page "around it" prepared to add/accept more testing information, so absolutely, our community can consider this page "Open For PR Business"  🚀

**Comment by bandantonio** [comment]
@sarah11918 I will take care of this guide, feel free to assign the issue to me

**Comment by sarah11918** [comment]
@bandantonio Done! Happy to answer any questions here or in the docs channel, but I think our initial discussion shows we're on the same page. So, looking forward to see what you come up with!  🙌
